### PR TITLE
i created a navagation bar

### PR DESCRIPTION
**Pull request by MrFrostUg**
i created a navigation bar and used album templates for my index page layout

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  line-height: 1.4;
+}
+
+/* Style links in the footer */
+a {
+  color: #e03333;
+  text-decoration: none;
+}
+
+.Parent {
```
Comment (hi-coders) [inline style.css]: lower case!

File: style.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  display: flex;
+  flex-direction: row;
+  border: 5px solid;
+  padding: 10px;
+  grid-gap: 15px;
+}
+
+.child1 {
```
Comment (hi-coders) [inline style.css]: child1 child2 are not good names

File: style.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+}
+
+.child2 {
+  width: 50%;
+  background-color: aqua;
+  color: green;
+  height: 50vh;
+};
+.navbar-logo {
+width: 10px;
```
Comment (hi-coders) [inline style.css]: use indentation properly

**Review by hi-coders — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (MrFrostUg) [reply style.css]: resolved to parent, left and right grid respectively

File: style.css (hunk 1 above)
Comment (MrFrostUg) [reply style.css]: i have not understood by what you mean here please, what should i make lower case please?

File: style.css (hunk 3 above)
Comment (MrFrostUg) [reply style.css]: resolved by formatting using the respective default language formatt

**Comment by MrFrostUg** [comment]
navigation bar added and cards to the home page

**Pull request closed without merging**